### ISSUE TITLE
CSS minification breaks calc()

### ISSUE THREAD

**Issue by girlie** [wp-media/wp-rocket#1554]
Example code to test:

```
background-position:
calc(100% - 20px) calc(1em + 2px),
calc(100% - 15px) calc(1em + 2px),
calc(100% - 2.5em) 0.5em;
```

Result after minification (space removed around plus signs):

`background-position:calc(100% - 20px) calc(1em+2px),calc(100% - 15px) calc(1em+2px),calc(100% - 2.5em) .5em`

_Reference: https://secure.helpscout.net/conversation/788655658/98351/_

**Comment by Tabrisrp** [comment]
https://github.com/matthiasmullie/minify/issues/274

**Comment by Tabrisrp** [comment]
Associated PR https://github.com/matthiasmullie/minify/pull/275

**Comment by kieran-wpmedia** [comment]
Related ticket here: https://secure.helpscout.net/conversation/962376555/123955/

Original:
`.example-with-semi-colon {
  width: calc(100% + 1px);
}

.example-without-semi-colon {
  width: calc(100% + 1px)
}`

After minification:
`.example-with-semi-colon{width:calc(100% + 1px)}.example-without-semi-colon{width:calc(100%+1px)}`

**Comment by Tabrisrp** [comment]
This is fixed in v1.3.63 of the Minify library, we can include the update in 3.5

**Comment by kierantaylorio** [comment]
Related ticket: https://secure.helpscout.net/conversation/1087609903/146884
